regulations/static/regulations/js/source/views/preamble-view.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var ChildView = require('./main/child-view');
+var CommentView = require('./comment-view');
+
+var PreambleView = ChildView.extend({
+  el: '#content-wrapper',
```

Comment:
This might be a helpful change for clarity, but I don't think we'd need it for functionality. `MainView` checks the page type of the top-level section, then instantiates a child view of the corresponding type:

```javascript

    viewmap: {
        'reg-section': RegView,
        'landing-page': RegView,
        'search-results': SearchResultsView,
        'diff': DiffView,
        'appendix': RegView,
        'interpretation': RegView,
        'comment-review': CommentReviewView,
        'preamble-section': PreambleView
    }
```

Meaning that any given page should only have a single element with `id="content-wrapper"` (which makes sense, given that we're using an ID), and that we only instantiate one of the keys in `viewmap` at once.